lib/index.js

```diff
@@ -332,21 +345,29 @@ function _validateReceiveHeader(header, value) {
   }
 }
 
-function _signingString(
-  {includeHeaders, headers, method, url, host, path, validate}) {
+function _signingString({includeHeaders, headers, method, url,
+  host, path, requestOptions, validate}) {
   const result = [];
+  const addValue = (key, values, label) => {
```

Comment:
This looks like it could be a separate helper (`_addValue(result, key, values, label)`) so it's not created each time we call `_signingString`.

Reply:
Addressed in https://github.com/digitalbazaar/http-signature-header/pull/12/commits/bec870709f6558367a7882308bb1b81094cc673c